src/test/e2e/common.go

```diff
@@ -27,6 +28,10 @@ func GetCLIName() string {
 		} else {
 			binaryName = "zarf-mac-intel"
 		}
+	} else if runtime.GOOS == "windows" {
```

Comment:
Just repeating the comment about supporting Windows on ARM here as well.

Reply:
FWIW I agree on WoA.  We should support if we can.